src/main/resources/org/jenkinsci/plugins/workflow/multibranch/Messages.properties

```diff
@@ -1,3 +1,7 @@
 ReadTrustedStep._has_been_modified_in_an_untrusted_revis=\u2018{0}\u2019 has been modified in an untrusted revision
 WorkflowMultiBranchProject.DisplayName=Multibranch Pipeline
 WorkflowMultiBranchProject.Description=Creates a set of Pipeline projects according to detected branches in one SCM repository.
+JobPropertyStep._could_remove_warning=WARNING: The 'properties' step will remove all 'JobProperty's currently configured in this job, \
+  either from the UI or from an earlier 'properties' step.\n\
+  This includes configuration for discarding old builds, parameters, concurrent builds and build triggers.
+JobPropertyStep._removed_property_warning=WARNING: Removing existing job property ''{0}''
```

Comment:
Save yourself some sleepness nights wondering whether your `MessageFormat` is correctly escaping metacharacters, and use typographically correct quotes which do not suffer from this issue to begin with. Here’s a nickel kid, get yourself a “‘’”